### ISSUE TITLE
rpm spec file

### ISSUE THREAD

**Issue by nerijus** [asbru-cm/asbru-cm#24]
I'd suggest making these changes:
```
--- asbru.spec.1        2017-11-21 01:20:31.718597897 +0200
+++ asbru.spec  2017-11-07 19:22:21.818593285 +0200
@@ -97,7 +58,7 @@
 mkdir -p %{buildroot}/{%{_mandir}/man1,%{_bindir}}
 mkdir -p %{buildroot}/%{_datadir}/{%{name}/{lib,res},applications}
 mkdir -p %{buildroot}/%{_bashcompletiondir}
-mkdir -p %{buildroot}/%{_datadir}/icons/hicolor/{24x24,64x64}/apps
+mkdir -p %{buildroot}/%{_datadir}/icons/hicolor/{24x24,64x64,256x256,scalable}/apps
 
 install -m 755 asbru %{buildroot}/%{_bindir}/%{name}
 install -m 755 utils/pac_from_mcm.pl %{buildroot}/%{_bindir}/%{name}_from_mcm
@@ -110,6 +71,8 @@
 # Copy the icons over to /usr/share/icons/
 cp -a res/asbru-logo-24.png %{buildroot}/%{_datadir}/icons/hicolor/24x24/apps/%{name}.png
 cp -a res/asbru-logo-64.png %{buildroot}/%{_datadir}/icons/hicolor/64x64/apps/%{name}.png
+cp -a res/asbru-logo-256.png %{buildroot}/%{_datadir}/icons/hicolor/256x256/apps/%{name}.png
+cp -a res/asbru-logo.svg %{buildroot}/%{_datadir}/icons/hicolor/scalable/apps/%{name}.svg
 
 # Copy the remaining resources and libraries
 cp -a res/*.{png,jpg,pl,glade} res/termcap %{buildroot}/%{_datadir}/%{name}/res/
```

**Comment by nerijus** [comment]
Instead of
Icon=/opt/asbru/res/asbru-logo-256.png
I suggest using
Icon=asbru
in the desktop file
then it will automatically use the best icon.

**Comment by nerijus** [comment]
Also instead of using Exec=/opt/asbru/asbru I suggest using Exec=asbru-cm in the desktop file. Besides, in the rpm package there is no /opt/asbru at all.

**Comment by KlaasT** [comment]
Going to have a look at that. There is some work to do for the .deb packages as well.

**Comment by KlaasT** [comment]
I've added your patch to the spec file. Thanks for that. About the desktop file I will need to look a bit further as we shouldn't break it for other distributions.

**Comment by gfrenoy** [comment]
The desktop file is somehow related to Gnome so should not be distribution-specific, shouldn't it ?

**Comment by nerijus** [comment]
Of course, so my changes should be good. Anyway, if they are not, you will get bug reports, but I don't think there will be any.

**Comment by KlaasT** [comment]
I've edited the .desktop file. We should be able to test it using the latest snapshot which should be built in a few minutes.

**Comment by gfrenoy** [comment]
Just tested the latest snapshot (great that automatic build !!!) on Fedora but it does not work yet.

What I found out (but not yet fixed...):

- executable is "asbru-cm" on Fedora but "asbru" on other distribution, we need to decide on which one we want to use and keep the same for all distros
- there are still references to /opt/asbru but on Fedora, application is installed on /usr/share/asbru

To be continued ... PRs are welcome :-)

**Comment by cornfeedhobo** [comment]
I like what you are doing here. In case this helps, [this is what I put together](https://build.opensuse.org/package/view_file/home:cornfeedhobo:devel:languages:perl/asbru-cm/asbru-cm.spec?expand=1)

This was built for a KDE system. Of note is the removal of `perl(Gtk:Appindicator)` which was causing a system tray icon unresponsive to left clicks.

**Comment by gfrenoy** [comment]
The new spec file is available in our [5.0.0 release](https://github.com/asbru-cm/asbru-cm/releases/tag/5.0.0).
